app/views/spree/admin/webhooks_subscribers/show.html.erb

```diff
@@ -0,0 +1,61 @@
+<% content_for :page_title do %>
+  <%= link_to Spree.t(:webhooks_subscribers), spree.admin_webhooks_subscribers_path %> /
+  <%= Spree.t(:webhook_events) %>
+<% end %>
+
+<fieldset>
+  <div class="table-responsive">
+    <table class="table table-bordered">
+      <tr>
+        <td><strong><%= Spree.t(:url) %></strong></td>
+        <td><%= @webhooks_subscriber.url %></td>
+      </tr>
+      <tr>
+        <td><strong><%= Spree.t(:status) %></strong></td>
+        <td><%= @webhooks_subscriber.active ? Spree.t(:active) : Spree.t(:inactive) %></td>
+      </tr>
+      <tr>
+        <td><strong><%= Spree.t(:subscriptions) %></strong></td>
+        <td><%= @webhooks_subscriber.subscriptions&.join(', ') %></td>
+      </tr>
+    </table>
+  </div>
+</fieldset>
+
+<% events = @webhooks_subscriber.events.order(created_at: :desc).page(params[:page]).per(params[:per_page]) %>
+
+<% if events.any? %>
+  <div class="table-responsive">
+    <table class="table">
+      <thead>
+      <tr>
+        <th><%= Spree.t(:name) %></th>
+        <th><%= Spree.t(:successful) %></th>
+        <th><%= Spree.t(:created_at) %></th>
+        <th><%= Spree.t(:response_code) %></th>
+        <th><%= Spree.t(:execution_time) %></th>
+        <th><%= Spree.t(:request_errors) %></th>
+        <th></th>
+      </tr>
+      </thead>
+      <tbody>
+      <% events.each do |event| %>
+        <tr id="<%= spree_dom_id event %>">
+          <td><%= event.name %></td>
+          <td><%= event.success ? Spree.t(:true) : Spree.t(:false) %></td>
+          <td><%= event.created_at %></td>
+          <td><%= event.response_code %></td>
+          <td><%= event.execution_time %></td>
+          <td><%= event.request_errors.gsub('[SPREE WEBHOOKS]', '') %></td>
```

Comment:
@aleksandarpetrushev is that output sanitized here?

Reply:
<img width="410" alt="Screenshot 2021-11-30 at 12 03 33" src="https://user-images.githubusercontent.com/20384428/144035907-0f5ba1b2-4080-48b7-ace2-29fea4a10c69.png">
This is how it looks

Reply:
Should we remove the for #{url} part?
@damianlegawiec

Reply:
@aleksandarpetrushev I'm rather asking how are we treating the response we're receiving from the webhook page, are we sanitizing it? eg. some malicious platform can return some XSS